python/py-gwdatafind/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-gwdatafind
+version             1.0.0
+
+categories-append   science
+maintainers         {ligo.org:duncan.macleod @duncanmmacleod} openmaintainer
+
+platforms           darwin
+license             GPL-3
+
+description         The client library for the LIGO Data Replicator (LDR) \
+                    service.
+long_description    The DataFind service allows users to query for the \
+                    location of Gravitational-Wave Frame (GWF) files \
+                    containing data from the current gravitational-wave \
+                    detectors.
+homepage            https://gwdatafind.readthedocs.io
+
+master_sites        pypi:g/gwdatafind
+distname            gwdatafind-${version}
+checksums   rmd160  d6ee9ee4ab579341543458acef32b6a12cf6d628 \
+            sha256  1441ad9dc445fc906bd3f589a059b878976958870cfad9ef20b9e2b8255378f4 \
+            size    31273
+
+python.versions     27 36
+python.default_version 36
+
+if {${name} ne ${subport}} {
+
+    depends_lib-append  port:py${python.version}-six \
+                        port:py${python.version}-ligo-segments \
+                        port:py${python.version}-openssl
+
+    depends_build-append port:py${python.version}-setuptools
+
+}
```

Comment:
perhaps add this: 
```
livecheck.type      none
} else {
    livecheck.type      pypi
}
```
so that it will disable the livecheck for the subports and only do it for ```py-gwdatafind```; that's how it is done for most others.